libraries/abstractions/cellular/test/test_cellular_api.c

```diff
@@ -283,7 +286,8 @@ static void prvGenericCallback( const char * pRawData )
 
 /* PDN event callback function. */
 static void prvPdnEventCallback( CellularUrcEvent_t urcEvent,
-                                 uint8_t contextId )
+                                 uint8_t contextId,
```

Comment:
should we add const here?

Reply:
This is not modified in the commit. I suggest we have another commit review over the test_cellular_api.c.